src/cmdline.c

```diff
@@ -90,7 +90,12 @@ int cmdline_run(const char **argv, int argc)
 
 	argparse_t argparse;
 	argparse_init(&argparse, options, usage, 0);
-	argc = argparse_parse(&argparse, argc, argv);
+	const char** out = malloc(sizeof(char**) * argc);
+	for (size_t i = 0; i < argc; i++)
```

Comment:
You can avoid the loop by using [`memcpy`](http://www.cplusplus.com/reference/cstring/memcpy/)